src/copy.jl

```diff
@@ -19,6 +19,51 @@ Base.delete!(idxmap::IndexMap, ci::MOI.ConstraintIndex) = delete!(idxmap.conmap,
 
 Base.keys(idxmap::IndexMap) = Iterators.flatten((keys(idxmap.varmap), keys(idxmap.conmap)))
 
+"""
+    copyattributes!(dest::MOI.AbstractInstance, src::MOI.AbstractInstance, idxmap::IndexMap, cancopyattr::Function=MOI.canset, copyattr!::Function=MOI.set!)
+
+Copy the instance attributes from the instance `src` the instance `dest` using `cancopyattr` to check if the attribute can be copied and `copyattr!` to copy the attribute.
+
+    copyattributes!(dest::MOI.AbstractInstance, src::MOI.AbstractInstance, idxmap::IndexMap, vis_src::Vector{MOI.VariableIndex}, cancopyattr::Function=MOI.canset, copyattr!::Function=MOI.set!)
+
+Copy the variable attributes from the instance `src` the instance `dest` using `cancopyattr` to check if the attribute can be copied and `copyattr!` to copy the attribute.
+
+    copyattributes!(dest::MOI.AbstractInstance, src::MOI.AbstractInstance, idxmap::IndexMap, cis_src::Vector{MOI.ConstraintIndex{F, S}}, cancopyattr::Function=MOI.canset, copyattr!::Function=MOI.set!) where {F, S}
+
+Copy the constraint attributes of `F`-in-`S` constraints from the instance `src` the instance `dest` using `cancopyattr` to check if the attribute can be copied and `copyattr!` to copy the attribute.
+"""
+function copyattributes! end
+
+function copyattributes!(dest::MOI.AbstractInstance, src::MOI.AbstractInstance, idxmap::IndexMap, cancopyattr::Function=MOI.canset, copyattr!::Function=MOI.set!)
```

Comment:
Using the name "copy" here is confusing given that these methods are also used for 2-pass IO.

Reply:
What about "passattributes" ?

Reply:
Better